core/src/main/java/com/exadel/etoolbox/linkinspector/api/Status.java

```diff
@@ -14,40 +14,33 @@
 
 package com.exadel.etoolbox.linkinspector.api;
 
+import lombok.Getter;
 import org.apache.http.HttpStatus;
 import org.apache.commons.lang3.StringUtils;
 import org.apache.commons.lang3.builder.EqualsBuilder;
 import org.apache.commons.lang3.builder.HashCodeBuilder;
 
 /**
- * Represents the of the {@link Link} based on a result of checking link's validity
+ * Represents the of the {@link Result} based on a result of checking link's validity
  */
-public final class LinkStatus {
+@Getter
```

Comment:
Can be replace with @AllArgsConstructor and @EqualsAndHashCode, final must be removed for int code and String message.